packages/xo-web/src/common/tooltip/index.js

```diff
@@ -166,3 +166,8 @@ export default class Tooltip extends Component {
     return children
   }
 }
+
+// ===================================================================
+
+export const conditionalTooltip = (component, tooltip) =>
+  tooltip ? <Tooltip content={tooltip}>{component}</Tooltip> : component
```

Comment:
Explicit test. We might want `''` to be handled as "no tooltip" but we definitely want `0` to be shown.